frontend/src/components/create-contract/CreateContractForm.tsx

```diff
@@ -6,99 +6,197 @@ import { ListBox } from '../ListBox';
+
+  const tokenName = process.env.REACT_APP_PAYMENT_TOKEN_NAME;
+
   return (
-    <div className="mt-6 overflow-hidden shadow sm:rounded-md">
-      <div className="bg-white px-4 py-5 sm:p-6">
-        <div className="grid grid-cols-6 gap-6">
-          <div className="col-span-6">
-            <Input id="title" label="Title" />
-          </div>
-          <div className="col-span-6">
-            <TextArea id="description" rows={10} label="Description" />
-          </div>
-          <div className="col-span-6">
-            <TextArea
-              id="acceptance-criteria"
-              label="Acceptance Criteria"
-              rows={5}
-            />
-          </div>
-          <div className="col-span-2">
-            <label className="block text-sm font-medium text-gray-700">
-              Labels
-            </label>
-            <ListBox items={labelItems}></ListBox>
-          </div>
-          <div className="col-span-2">
-            <label className="block text-sm font-medium text-gray-700">
-              Identity
-            </label>
-            <ListBox items={identityItems}></ListBox>
-          </div>
-          <div className="col-span-2">
-            <label className="block text-sm font-medium text-gray-700">
-              Acceptance Tests
-            </label>
-            <ListBox items={acceptanceTestsItems}></ListBox>
-          </div>
-          <div className="col-span-2">
-            <label
-              htmlFor="bounty"
-              className="block text-sm font-medium text-gray-700"
-            >
-              Bounty
-            </label>
-            <Currency id="bounty" placeholder="1" label="ETH" />
-          </div>
-          <div className="col-span-2">
-            <label
-              htmlFor="buy-in"
-              className="block text-sm font-medium text-gray-700"
-            >
-              Buy-In
-            </label>
-            <Currency id="buy-in" placeholder="0.3" label="ETH" />
-          </div>
-          <div className="col-span-2">
-            <label
-              htmlFor="end-date"
-              className="block text-sm font-medium text-gray-700"
-            >
-              End Date
-            </label>
-            <input
-              type="date"
-              id="end-date"
-              className="mt-1 block w-full rounded-md border-gray-300 shadow-sm focus:border-indigo-500 focus:ring-indigo-500 sm:text-sm"
-            />
-          </div>
-          <div className="col-span-6">
-            <label className="block text-sm font-medium text-gray-700">
-              Attach Files
-            </label>
-            <Dropzone />
-          </div>
-          <div className="col-span-6 flex w-full bg-gray-50 px-4 py-5 sm:p-6">
-            <div className="w-1/2">
-              <PaymentSummary />
-            </div>
-            <div className="w-1/2">
-              <OptionsSummary />
+    <Formik
+      initialValues={initialValues}
+      validate={(values) => {
```

Comment:
I'd just remove this validate prop for now, unless you want to move some of the validation logic in here. I think it's fine to do that in another PR though, to keep this one moving.